### PR TITLE
Drivers: Added handling of zero packet length in driver vmxnet3

### PR DESCRIPTION
**Pull request by KristianJerpetjon**

### CODE REVIEW

**Comment by staiyeba** [comment]
jenkins test please

**Comment by staiyeba** [comment]
jenkins test please

File: src/drivers/vmxnet3.cpp [hunk 1]
```diff
@@ -514,6 +514,19 @@ bool vmxnet3::receive_handler(const int Q)
...
+    
+    // Handle case of empty packet
+    if (UNLIKELY((comp.len & (VMXNET3_MAX_BUFFER_LEN-1)) == 0)) {
+      //TODO assert / log if eop and sop are not set in empty packet.
+
+      //release unused buffer
+      auto* packet = (net::Packet*) (tx.buffers[desc] - DRIVER_OFFSET - sizeof(net::Packet));
+      delete packet; // call deleter on Packet to release it
+      rx[Q].buffers[desc] = nullptr;
+
```
Comment (alfred-bratterud) [inline src/drivers/vmxnet3.cpp]: @KristianJerpetjon we need a stat for dropped packets here. Otherwise lgtm

**Comment by alfred-bratterud** [comment]
jenkins test please

**Pull request merged**